### ISSUE TITLE
How do the order properties of tasks and of task-list-memberships exactly work?  

### ISSUE THREAD

**Issue by PeterKuhnle** [awork-io/awork#72]
### What is your question?

Hi Api-Support,
Both order properties (within the task model and "task-list-membership")  are of type double and thus indicate that something like a floating-point representation of a date-time value would allow to control the task's order within lists or natural listings.  
Have tried to put there some values, high to low, low to high, integers only, etc.... Could not discover any correlation with according listings in the frontend. Is there fürther documentation existing or do I have to follow some "hidden" rules (f.e. using strict monotonous integer sequences)? 
Thanks in advance &  greets, Peter

**Comment by vdurante** [comment]
@PeterKuhnle the idea is that if you have 2 tasks, you can order them 1 and 2. Then, if a third is created between them, it can be ordered at 1.5.

I could double check for you if using high values (such as a timestamp) would work as well on the backend logic.

The frontend should respect the ordering of this order field, except in some views that may have some type of ordering applied. Could you provide some example so we can understand what is going on?

**Comment by PeterKuhnle** [comment]
Hi Vitor,
let's assume you want to order a list of 100 task according their CreationDate. The easiest way would be to use a fractional representation of the timestamp (f.e. days since 1900-00-00 + MinuteOfDay/1440). That would give me a deterministic order. New-to-old could be the reciprocal value. I have not succeeded with that approach, whyever. 
Now I've started to iterate the ordered task-list on client side and just incrementing order starting from 1. Here I've discovered that the given order-number is not applied, but somehow interpreted and a modified order is returned. 
So, it would be nice to know, how the posted order-number is processed.
